src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] It is required that your `App.js` component is also class-based. Kindly change it from a functional component to a class-based one just as you did for the `Calculator.js component.

Reply:
thanks this has been done